includes/CMB2_Sanitize.php

```diff
@@ -212,10 +225,12 @@ public function text_datetime_timestamp( $repeat = false ) {
 			return $repeat_value;
 		}
 
-		$this->value = strtotime( $this->value['date'] . ' ' . $this->value['time'] );
+		if ( is_array( $this->value ) && isset( $this->value['date'] ) ) {
```

Comment:
Does this account for repeating datetime fields? 

I'm seeing this issue mentioned in #330 as well. I was looking through the code and believe I have narrowed it down to the deleted line 215 failing to account for repeatable fields. However I havnt had the time to try to fix it.